protoc-gen-eventuals/templates/eventuals-client.h.j2

```diff
@@ -0,0 +1,85 @@
+#pragma once
+{%
+set namespaces = package_name.split('.')
+-%}
+#include "eventuals/grpc/client.h"
+
+#include "{{ pb_header }}"
+#include "{{ eventuals_header }}"
+#include "eventuals/finally.h"
+#include "eventuals/head.h"
+#include "eventuals/let.h"
+#include "eventuals/then.h"
+#include "eventuals/do-all.h"
+#include "gtest/gtest.h"
+
+#include <grpcpp/channel.h>
+#include <string>
+
+using eventuals::DoAll;
+using eventuals::Finally;
+using eventuals::Head;
+using eventuals::Let;
+using eventuals::Then;
+
+using grpc::Status;
+
+using eventuals::grpc::Stream;
+using eventuals::grpc::Client;
+using eventuals::grpc::CompletionPool;
+using stout::Borrowable;
+using namespace {{ namespaces | join('::') }}::eventuals;
+
+{% for service in services %}
+class {{ service.name }}Client {
+public:
+
+  {{ service.name }}Client(
+      const std::string& target,
+      const std::shared_ptr<::grpc::ChannelCredentials>& credentials,
+      stout::borrowed_ref<CompletionPool> pool) : client_(target, credentials, std::move(pool)) {}
```

Comment:
```suggestion
      stout::borrowed_ref<CompletionPool> pool) : client_(target, credentials, std::move(pool)) = default;
```

Reply:
this can't be defaulted, since we call non default `client_` constructor